vars/simplePipeline.groovy

```diff
@@ -30,7 +30,7 @@ def call(Map pipelineParams) {
             stage('OWASP Dependency check') {
                 steps {
                     container('dependency-check') {
-                        sh './dependency-check.sh > /dev/null'
```

Comment:
IIRC the stdout output from the dependency checker was redirected to /dev/null so as to avoid logging potentially sensitive vulnerability information in the jenkins build log. Why has the redirection been removed?

Reply:
Sorry the dependency check stopped working whilst I was in a Lakeland cottage with poor internet. Wanted to understand what was failing